src/test/resources/projects/Standard8Project1EnvironmentVariables/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You can make these test data file smaller if all they're used for is to test the environment variables.